lib/session.js

```diff
@@ -78,6 +80,14 @@ class Session {
     return this.config.readme;
   }
 
+  get waitTimeSingleApproval() {
+    return this.config.waitTimeSingleApproval;
```

Comment:
Umm, on a second thought, why are the defaults set in the PR checker, instead of here?

Reply:
Ah yes, that makes more sense. I'll move it here.

Reply:
On second thought, this won't work because `git node metadata` doesn't use a Session. So `git node land` would get the defaults correctly, but `git node metadata` wouldn't.

We could create a MetadataSession, but it seems a bit too much. Also doesn't make sense since the idea of a session (to my understanding at least) is to save the state of a command to be resumed later.

Should we keep this here then?

Reply:
ping @joyeecheung

Reply:
Ah, makes sense, I forgot that the PR checker is also run by the metadata generator.